mappings/net/minecraft/client/render/entity/model/RaftModel.mapping

```diff
@@ -0,0 +1,19 @@
+CLASS net/minecraft/unmapped/C_eqfsmnzb net/minecraft/client/render/entity/model/RaftModel
+	FIELD f_bsvltcmi RIGHT_PADDLE Ljava/lang/String;
+	FIELD f_cfhhirmb LEFT_PADDLE Ljava/lang/String;
+	FIELD f_drqqjxrd rightPaddle Lnet/minecraft/unmapped/C_rglqxnbw;
+	FIELD f_fmictjks root Lcom/google/common/collect/ImmutableList;
+	FIELD f_gjbwdqbo leftPaddle Lnet/minecraft/unmapped/C_rglqxnbw;
```

Comment:
Doesn't make an awful lot of sense for the strings to be named the same as whatever C_rglqxnbw is. Are they translation keys or what?

Reply:
`C_rglqxnbw` is `ModelPart`, so yeah, its what part of the model that is being stored